<ui version="4.0" >
 <class>CreateNewEvent</class>
 <widget class="QDialog" name="CreateNewEvent" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>572</width>
    <height>339</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle" >
   <string>Create new event</string>
  </property>
  <widget class="QWidget" name="layoutWidget" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>10</y>
     <width>554</width>
     <height>320</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" >
    <item>
     <layout class="QHBoxLayout" >
      <item>
       <layout class="QVBoxLayout" >
        <item>
         <widget class="QLabel" name="label" >
          <property name="text" >
           <string>Ca&amp;lendar:</string>
          </property>
          <property name="buddy" >
           <cstring>selectCalendarBox</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelEventName" >
          <property name="text" >
           <string>&amp;Event Name:</string>
          </property>
          <property name="buddy" >
           <cstring>eventNameInput</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelDescription" >
          <property name="text" >
           <string>&amp;Description:</string>
          </property>
          <property name="buddy" >
           <cstring>textEdit</cstring>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" >
        <item>
         <layout class="QHBoxLayout" >
          <item>
           <widget class="QComboBox" name="selectCalendarBox" >
            <property name="sizePolicy" >
             <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize" >
             <size>
              <width>400</width>
              <height>0</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <spacer>
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType" >
             <enum>QSizePolicy::MinimumExpanding</enum>
            </property>
            <property name="sizeHint" >
             <size>
              <width>61</width>
              <height>24</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" >
          <item>
           <widget class="QLineEdit" name="eventNameInput" >
            <property name="sizePolicy" >
             <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize" >
             <size>
              <width>400</width>
              <height>0</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <spacer>
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType" >
             <enum>QSizePolicy::MinimumExpanding</enum>
            </property>
            <property name="sizeHint" >
             <size>
              <width>61</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType" >
           <enum>QSizePolicy::MinimumExpanding</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>461</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTextEdit" name="textEdit" >
      <property name="sizePolicy" >
       <sizepolicy vsizetype="Maximum" hsizetype="Maximum" >
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize" >
       <size>
        <width>550</width>
        <height>150</height>
       </size>
      </property>
      <property name="html" >
       <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Lucida Grande'; font-size:9pt; font-weight:400; font-style:normal;">
&lt;p style="-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;/p>&lt;/body>&lt;/html></string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" >
      <item>
       <layout class="QHBoxLayout" >
        <item>
         <widget class="QLabel" name="labelDataEdit" >
          <property name="text" >
           <string>D&amp;ate:</string>
          </property>
          <property name="buddy" >
           <cstring>dateEdit</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDateEdit" name="dateEdit" />
        </item>
       </layout>
      </item>
      <item>
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType" >
         <enum>QSizePolicy::MinimumExpanding</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>151</width>
          <height>32</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" >
        <item>
         <widget class="QLabel" name="labelTime1" >
          <property name="text" >
           <string>Time: From</string>
          </property>
          <property name="buddy" >
           <cstring>timeedit1</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="TimeEdit" name="timeedit1" />
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" >
        <item>
         <widget class="QLabel" name="labelTime2" >
          <property name="text" >
           <string>To</string>
          </property>
          <property name="buddy" >
           <cstring>timeedit2</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="TimeEdit" name="timeedit2" />
        </item>
       </layout>
      </item>
      <item>
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType" >
         <enum>QSizePolicy::Minimum</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>5</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" >
      <item>
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType" >
         <enum>QSizePolicy::MinimumExpanding</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="createButton" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text" >
         <string>C&amp;reate</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="cancelButton" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text" >
         <string>&amp;Cancel</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType" >
         <enum>QSizePolicy::Minimum</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>5</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TimeEdit</class>
   <extends>QTimeEdit</extends>
   <header>TimeEdit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>selectCalendarBox</tabstop>
  <tabstop>eventNameInput</tabstop>
  <tabstop>textEdit</tabstop>
  <tabstop>dateEdit</tabstop>
  <tabstop>timeedit1</tabstop>
  <tabstop>timeedit2</tabstop>
  <tabstop>createButton</tabstop>
  <tabstop>cancelButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>CreateNewEvent</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>523</x>
     <y>314</y>
    </hint>
    <hint type="destinationlabel" >
     <x>283</x>
     <y>306</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
